.githooks/pre-commit

```diff
@@ -4,7 +4,7 @@
 # are installed, and if so, uses the installed version to format
 # the staged changes.
 
-base=clang-format-10
+base=clang-format-12
```

Comment:
[Note]: This is leftover from pervious commit and may be moved to a separate PR if needed.